### ISSUE TITLE
SlimeVR 1.0 Release

### ISSUE THREAD

**Issue by TheButlah** [SlimeVR/SlimeVR-Server#577]
This is a tracking issue for us to discuss and track updates on features, bugs, and UX that we want in a SlimeVR 1.0 release. Our goal of this release is to have the software "shippable". We would be confident that we did everything we needed for reviewers and users to be happy, especially by nontechnical users.

Issues remaining for 1.0:
* Do a polish pass on sensible defaults for features
* Do a ux polish pass to clean any warts
* [ ] https://github.com/SlimeVR/SlimeVR-Server/issues/36
* [x] https://github.com/SlimeVR/SlimeVR-Server/issues/622
* [ ] https://github.com/SlimeVR/SlimeVR-Server/issues/446
* [ ] https://github.com/SlimeVR/SlimeVR-Server/issues/283
* [ ] https://github.com/SlimeVR/SlimeVR-Installer/issues/42
* [x] https://github.com/SlimeVR/SlimeVR-Server/issues/578
* [ ] https://github.com/SlimeVR/SlimeVR-Feeder-App/issues/8
* [x] https://github.com/SlimeVR/SlimeVR-Server/issues/511
* [ ] https://github.com/SlimeVR/SlimeVR-Feeder-App/issues/9
* [ ] https://github.com/SlimeVR/SlimeVR-Server/issues/579
* [x] https://github.com/SlimeVR/SlimeVR-Server/issues/74

Not needed for for 1.0:
* [x] https://github.com/SlimeVR/SlimeVR-Server/issues/405

Descoped due to blockers:
* [x] https://github.com/SlimeVR/SlimeVR-Server/issues/580 (Needs VRChat to release OSCQuery first)

Overlay issues, should we choose to include it in 1.0:
* [x] https://github.com/SlimeVR/SlimeVR-Rust/issues/81
* [ ] https://github.com/SlimeVR/SlimeVR-Rust/issues/186
* [ ] https://github.com/SlimeVR/SlimeVR-Rust/issues/80
* [ ] https://github.com/SlimeVR/SlimeVR-Rust/issues/185

**Comment by Louka3000** [comment]
> OSC can use legtweaks and the other goodies that SteamVR has
Can just say "Support OSCQuery for VRChat"

**Comment by ButterscotchV** [comment]
> Autobone UX done

AutoBone GUI needs a bit of a rework from its current state as in issue #446 and #283. Basically just needs to show the movements all before starting the recording and needs a proper finished visual guide for the movements.

**Comment by ButterscotchV** [comment]
I think these issues should also be included in 1.0 or clarified/updated to fit it:
- https://github.com/SlimeVR/SlimeVR-Server/issues/405
- https://github.com/SlimeVR/SlimeVR-Server/issues/75

**Comment by TheButlah** [comment]
> I think these issues should also be included in 1.0 or clarified/updated to fit it:
> 
>     * [[Server] Upload firmware via OTA #405](https://github.com/SlimeVR/SlimeVR-Server/issues/405)
> 
>     * [[Server] Building firmware by server #75](https://github.com/SlimeVR/SlimeVR-Server/issues/75)

Eiren mentioned in #75 that its actually not necessary for 1.0, so I've updated the issue to clarify.

**Comment by TheButlah** [comment]
Descoped OSCQuery due to conversations in #580

**Comment by ImUrX** [comment]
is there a tracking issue for having the tracker handling more than one server in the network?

**Comment by Eirenliel** [comment]
#622 is required for 1.0.

**Comment by TheButlah** [comment]
We have so much that is blocking 1.0 ;(

**Comment by Eirenliel** [comment]
> We have so much that is blocking 1.0 ;(

This is not how it works.1.0 is not a feature that we all wait when it can be implemented to be blocked. We work forward and we will get there eventually.

**Comment by TheButlah** [comment]
IK, was making an observation that there is *a lot* that should be done before reviewers have this in hand. At the current pace of things, it seems unrealistic to assume everything on this list will happen before reviewers review.

**Comment by Eirenliel** [comment]
I love how this has nothing that we *actually* need for 1.0 or have done for 1.0 😢
